### ISSUE TITLE
GS1 FNC1 symbol to datamatrix

### ISSUE THREAD

**Issue by xjpf** [dmtx/libdmtx#58]
Hi. Keep waiting for possibility to add FNC1 symbol to datamatrix when generating image! :)

**Comment by msva** [comment]
wasn't it implemented in #10 and merged long time ago?

**Comment by otonoton** [comment]
Yes it was, I have been using it quite successfully via KDE-Prison.

**Comment by msva** [comment]
thanks for confirmation

**Comment by Yser2025** [comment]
> wasn't it implemented in [#10](https://github.com/dmtx/libdmtx/pull/10) and merged long time ago?

Hello!
But some codes have few unprintable chars [for example structure](https://www.cleverence.ru/support/77127/):
`<FNC1>010465007519501721Z'%+*r_(()"'L<GS>91TEST<GS>92dGVzdARbyfJOm1B1r1+Srw7F3Xc3hRKHDPUUcaGh2k= `
`<FNC1> - 0xE8`
`<GS> - 0x1D`

Maybe we can make specification table for replacements? For exceptionally precise coding like
```
0xE8 to 0x7E
0x1D to 0x7C
...
```
For decode and back if encode?

Or, maybe mode to decode/encode with hex data? Without encoding to printable chars.
